apps/Standalone/src/templates/components/DevToolbox.tsx

```diff
@@ -34,6 +35,7 @@ export const DevToolbox = () => {
     [dispatch]
   );
 
+  const isLocal = useIsLocal();
```

Comment:
There is no local scenario, this is only azure specific, you can revert changes in this file completely.